content/docs/SUSHI/running/_index.md

```diff
@@ -19,24 +19,84 @@ SUSHI is executed from the command line. The general form of the SUSHI execution
 where options include the following (in any order):
 
 ```text
--o, --out <out>     the path to the output folder
 -d, --debug         output extra debugging information
+-h, --help          output usage information
+-i, --init          initialize a SUSHI project
+-o, --out <out>     the path to the **fsh-generated** folder
 -p, --preprocessed  output FSH produced by preprocessing steps
+--require-latest    indicate that SUSHI should halt if the latest SUSHI version is not installed
 -s, --snapshot      generate snapshot in Structure Definition output (default: false)
--i, --init          initialize a SUSHI project
 -v, --version       print SUSHI version
--h, --help          output usage information
 ```
```

Comment:
I see you've re-ordered this.  Maybe it doesn't matter, but the original order reflected how SUSHI actually prints out the usage:
<img width="707" alt="image" src="https://user-images.githubusercontent.com/2278253/173940773-89185cb6-59b5-4486-8997-389ba25613ad.png">

Reply:
OK, would you suggest we also update SUSHI to list them in alpha order then?